modules/nw-ovn-kubernetes-resources-con.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/ovn-kubernetes-architecture.adoc
+
+:_content-type: CONCEPT
+[id="nw-resources-ovn-kubernetes-project_{context}"]
+= Resources in the OVN-Kubernetes project
+
+The OVN-Kubernetes Container Network Interface (CNI) cluster network provider implements the following features:
+
+. Run the following command to get all resources in the OVN-Kubernetes project
++
+[source,terminal]
+----
+$ oc get all -n openshift-ovn-kubernetes
+----
```

Comment:
Concept modules don't have steps in them.

Reply:
I removed this section as covered elsewhere